### PR TITLE
[zh] Sync Wikimedia case-study

### PR DESCRIPTION
**Pull request by Sea-n**
It's updated to a new page template in 4d9ee76 (1.5 years ago)

### CODE REVIEW

File: content/zh/case-studies/wikimedia/index.html [hunk 1]
```diff
@@ -1,118 +1,151 @@
...
 cid: caseStudies
+
+new_case_study_styles: true
+heading_title_text: Wikimedia
+use_gradient_overlay: true
+subheading: >
+  利用 Kubernetes 构建工具提升世界的维基
+case_study_details:
+  - 公司：Wikimedia
+  - 地点：加州旧金山
```
Comment (howieyuen) [inline content/zh/case-studies/wikimedia/index.html:13]: 个人建议地点不翻译

File: content/zh/case-studies/wikimedia/index.html [hunk 2]
```diff
@@ -1,118 +1,151 @@
...
+case_study_details:
+  - Company: Wikimedia
+  - Location: San Francisco, CA
+-->
+
+<!--
+<p>The non-profit Wikimedia Foundation operates some of the largest collaboratively edited reference projects in the world, including Wikipedia. To help users maintain and use wikis, it runs Wikimedia Tool Labs, a hosting environment for community developers working on tools and bots to help editors and other volunteers do their work, including reducing vandalism. The community around Wikimedia Tool Labs began forming nearly 10 years ago.</p>
+-->
+<p>非营利的 Wikimedia 基金会运营着一些世界上最大的合作编辑参考项目，包括 Wikipedia。为了帮助用户维护和使用 wiki，它运行 Wikimedia 工具实验室，这是一个托管环境，为社区开发人员工作的工具和机器人，以帮助编辑和其他志愿者做他们的工作，包括减少破坏。Wikimedia 工具实验室周围的社区
+在近 10 年前开始形成。</p>
```
Comment (howieyuen) [inline content/zh/case-studies/wikimedia/index.html:34]: 合并 34 和 35 行

File: content/zh/case-studies/wikimedia/index.html [hunk 3]
```diff
@@ -1,118 +1,151 @@
...
+<br>
+"Wikimedia Tool Labs is vital for making sure wikis all around the world work as well as they possibly can. Because it's grown organically for almost 10 years, it has become an extremely challenging environment and difficult to maintain. It's like a big ball of mud — you really can't see through it. With Kubernetes, we're simplifying the environment and making it easier for developers to build the tools that make wikis run better."
+{{< /case-studies/quote >}}
+-->
+{{< case-studies/quote author="Yuvi Panda, Wikimedia 基金会和 Wikimedia 工具实验室的运维工程师">}}
+
+<img src="/images/wikimedia_logo.png" alt="Wikimedia">
+<br>
+<br>
+“Wikimedia 工具实验室对于确保世界各地的 wiki 尽可能工作至关重要。因为它有机地生长了近 10 年，它已成为一个极具挑战性的环境，难以维持。它就像一个大的泥球，你真的看不透它。借助 Kubernetes，可以简化环境，使开发人员能够更轻松地构建使 wiki 运行得更好的工具。”
```
Comment (howieyuen) [inline content/zh/case-studies/wikimedia/index.html:50]:
```suggestion
“Wikimedia 工具实验室对于确保世界各地的 wiki 尽可能正常运行至关重要。因为它有机地生长了近 10 年，所以它已成为一个极具挑战性且难以维护的环境。它就像一个大的泥球，你真的看不透它。借助 Kubernetes，我们正在简化环境并让开发人员更容易构建出使 wiki 更好运行的工具。”
```

File: content/zh/case-studies/wikimedia/index.html [hunk 4]
```diff
@@ -1,118 +1,151 @@
...
+
+<!--
+<h2>Using Kubernetes to provide tools for maintaining wikis</h2>
+-->
+<h2>使用 Kubernetes 提供维护 wiki 的工具</h2>
+
+<!--
+<p>Wikimedia Tool Labs is run by a staff of four-and-a-half paid employees and two volunteers. The infrastructure didn't make it easy or intuitive for developers to build bots and other tools to make wikis work more easily. Yuvi says, "It's incredibly chaotic. We have lots of Perl and Bash duct tape on top of it. Everything is super fragile."</p>
+-->
+<p>Wikimedia 工具实验室由四名半支薪员工和两名志愿者管理。基础架构无法使开发人员轻松或直观地构建机器人和其他工具，使 wiki 更易于工作。Yuvi 说，“它非常混乱，我们有很多的 Perl 和 Bash 缠绕在上面，一切都是超级脆弱。”</p>
```
Comment (howieyuen) [inline content/zh/case-studies/wikimedia/index.html:119]:
```suggestion
<p>Wikimedia 工具实验室由四个半带薪员工和两名志愿者管理。基础架构无法使开发人员轻松或直观地构建机器人和其他工具，使 wiki 更易于工作。Yuvi 说，“它非常混乱，我们有很多的 Perl 和 Bash 缠绕在上面，一切都是超级脆弱。”</p>
```

File: content/zh/case-studies/wikimedia/index.html [hunk 5]
```diff
@@ -1,118 +1,151 @@
...
+
+<!--
+<p>To solve the problem, Wikimedia Tool Labs migrated parts of its infrastructure to Kubernetes, in preparation for eventually moving its entire system. Yuvi said Kubernetes greatly simplifies maintenance. The goal is to allow developers creating bots and other tools to use whatever development methods they want, but make it easier for the Wikimedia Tool Labs to maintain the required infrastructure for hosting and sharing them.</p>
+-->
+<p>为了解决这个问题，Wikimedia 工具实验室将其部分基础设施迁移到了 Kubernetes，为最终迁移整个系统做准备。Yuvi 说，Kubernetes 大大简化了维护。目标是允许创建机器人和其他工具的开发人员使用他们想要的任何开发方法，但使 Wikimedia 工具实验室更容易维护托管和共享它们所
```
Comment (howieyuen) [inline content/zh/case-studies/wikimedia/index.html:124]: 合并 124 行 和 125 行

File: content/zh/case-studies/wikimedia/index.html [hunk 6]
```diff
@@ -1,118 +1,151 @@
...
+需的基础结构。</p>
+
+<!--
+<p>"With Kubernetes, I've been able to remove a lot of our custom-made code, which makes everything easier to maintain. Our users' code also runs in a more stable way than previously," says Yuvi.</p>
+-->
+<p>“借助 Kubernetes，我能够删除大量我们定制的代码，这使得所有内容更易于维护，我们的用户代码也以比以前更稳定的方式运行。” Yuvi 说。</p>
```
Comment (howieyuen) [inline content/zh/case-studies/wikimedia/index.html:130]:
```suggestion
<p>Yuvi 说：“借助 Kubernetes，我能够删除大量我们定制的代码，这使得所有内容更易于维护，我们的用户代码也以比以前更稳定的方式运行。” </p>
```

File: content/zh/case-studies/wikimedia/index.html [hunk 7]
```diff
@@ -1,118 +1,151 @@
...
+
+<!--
+<h2>Simplifying infrastructure and keeping wikis running better</h2>
+-->
+<h2>简化基础架构让 wiki 更好地运行</h2>
+
+<!--
+<p>Wikimedia Tool Labs has seen great success with the initial Kubernetes deployment. Old code is being simplified and eliminated, contributing developers don't have to change the way they write their tools and bots, and those tools and bots run in a more stable fashion than they have in the past. The paid staff and volunteers are able to better keep up with fixing issues.</p>
+-->
+<p>Wikimedia 工具实验室在最初的 Kubernetes 部署中取得了巨大成功。旧代码正在被简化和消除，使开发人员不必改变他们编写工具和机器人的方式，这些工具和机器人的运行方式比过去更稳定。付费员工和志愿者能够更好地解决问题。</p>
```
Comment (howieyuen) [inline content/zh/case-studies/wikimedia/index.html:140]:
```suggestion
<p>Wikimedia 工具实验室在最初的 Kubernetes 部署中取得了巨大成功。旧代码正在被简化和消除，使开发人员不必改变他们编写工具和机器人的方式，这些工具和机器人的运行方式比过去更稳定。带薪员工和志愿者能够更好地解决问题。</p>
```

File: content/zh/case-studies/wikimedia/index.html [hunk 8]
```diff
@@ -1,118 +1,151 @@
...
+
+<!--
+<p>In the future, with a more complete migration to Kubernetes, Wikimedia Tool Labs expects to make it even easier to host and maintain the bots and tools that help run wikis across the world. The tool labs already host approximately 1,300 tools and bots from 800 volunteers, with many more being submitted every day. Twenty percent of the tool labs' web tools that account for more than 60 percent of web traffic now run on Kubernetes. The tool labs has a 25-node cluster that keeps up with each new Kubernetes release. Many existing web tools are migrating to Kubernetes.</p>
+-->
+<p>将来，随着更完整的迁移到 Kubernetes，Wikimedia 工具实验室希望使托管和维护帮助在世界各地运行 wiki 的机器人和工具变得更加容易。该工具实验室已经托管了来自 800 名志愿者的大约 1300 个工具和机器人，每天提交更多工具和机器人。占 Web 流量 60% 以上的 20% 的工具实验
```
Comment (howieyuen) [inline content/zh/case-studies/wikimedia/index.html:145]: 合并 145 和 16 行

Comment (howieyuen) [inline content/zh/case-studies/wikimedia/index.html:145]:
```suggestion
<p>将来，随着更完整的迁移到 Kubernetes，Wikimedia 工具实验室希望更轻松地托管和维护有助于在世界各地运行 wiki 的机器人和工具。该工具实验室已经拥有来自 800 名志愿者的大约 1300 个工具和机器人，而且每天提交量会更多。占 Web 流量 60% 以上的工具实验室的 Web 工具中有 20% 现在运行在 Kubernetes 上。工具实验室有一个 25 节点的集群，可以跟上每个新的 Kubernetes 版本。许多现有的 Web 工具正在迁移到 Kubernetes。
```

File: content/zh/case-studies/wikimedia/index.html [hunk 9]
```diff
@@ -1,118 +1,151 @@
...
-                <!-- "Our goal is to make sure that people all over the world can share knowledge as easily as possible. Kubernetes helps with that, by making it easier for wikis everywhere to have the tools they need to thrive," says Yuvi. -->
-                “我们的目标是确保世界各地的人们能够尽可能轻松地分享知识。Kubernetes 通过让世界各地的 wiki 更容易拥有蓬勃发展所需的工具，从而帮助到您，” Yuvi 说。
-            </p>
-        </div>
-    </main>
-</section>
+<!--
+<p>"Our goal is to make sure that people all over the world can share knowledge as easily as possible. Kubernetes helps with that, by making it easier for wikis everywhere to have the tools they need to thrive," says Yuvi.</p>
+-->
+<p>“我们的目标是确保世界各地的人们能够尽可能轻松地分享知识，Kubernetes 可以帮助实现这一点，让各地的 wiki 更容易拥有蓬勃发展所需的工具。” Yuvi 说。</p>
```
Comment (howieyuen) [inline content/zh/case-studies/wikimedia/index.html:151]:
```suggestion
<p> Yuvi 说：“我们的目标是确保世界各地的人们能够尽可能轻松地分享知识，Kubernetes 帮助实现了这一点，它让世界各地的 wiki 更容易拥有蓬勃发展所需的工具。”</p>
```

**Review by howieyuen — CHANGES_REQUESTED** [review]

File: content/zh/case-studies/wikimedia/index.html (hunk 2 above)
Comment (Sea-n) [reply content/zh/case-studies/wikimedia/index.html:34]: Oh thanks! I didn't noticed that line break. 😅

File: content/zh/case-studies/wikimedia/index.html (hunk 9 above)
Comment (Sea-n) [reply content/zh/case-studies/wikimedia/index.html]: Thanks for those awesome suggestions! I've force-pushed the changes.

**Comment by howieyuen** [comment]
all checks are failed, I think it's ':' issue, we have to keep En punctuation rather than Zh in header section
```
8:44:23 PM: Error: Error building site: failed to render pages: render of "page" failed: "/opt/build/repo/layouts/case-studies/single.html:21:29": execute of template failed: template: case-studies/single.html:21:29: executing "main" at <$detail>: range can't iterate over 公司：Wikimedia
```

File: content/zh/case-studies/wikimedia/index.html [hunk 10]
```diff
@@ -1,118 +1,148 @@
...
+case_study_styles: true
 cid: caseStudies
+
+new_case_study_styles: true
+heading_title_text: Wikimedia
+use_gradient_overlay: true
+subheading: >
+  利用 Kubernetes 构建工具提升世界的维基
+case_study_details:
+  - 公司：Wikimedia
```
Comment (howieyuen) [inline content/zh/case-studies/wikimedia/index.html:12]: 看报错，这里应该是中英文冒号的问题

File: content/zh/case-studies/wikimedia/index.html (hunk 1 above)
Comment (howieyuen) [reply content/zh/case-studies/wikimedia/index.html:13]: 同上

**Review by howieyuen — CHANGES_REQUESTED** [review]

File: content/zh/case-studies/wikimedia/index.html (hunk 10 above)
Comment (Sea-n) [reply content/zh/case-studies/wikimedia/index.html:12]: Oh, I didn't realize this semi-comma is for YAML metadata! 😅

**Comment by Sea-n** [comment]
Thanks to @howieyuen's help, the CI is passed.

Preview: <https://deploy-preview-33759--kubernetes-io-main-staging.netlify.app/zh/case-studies/wikimedia/>

In [this comment](https://github.com/kubernetes/website/pull/33759#discussion_r874685868) we have different opinion about whether the location should be translated. How does @tengqm think?

**Pull request merged**